stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md

```diff
@@ -4,95 +4,114 @@ This package provides the functionality required to access and manipulate data s
 
-//Iterate the results-set and retrieve the first name.
-while (dt.hasNext()) {
-    ResultCustomers rs = check <ResultCustomers>dt.getNext();
-    firstName = rs.FIRSTNAME;
+```ballerina
+var ret1 = testDB->update("INSERT INTO Students(StudentID, LastName) values (1, 'john')");
+match ret1 {
+    int retInt => io:println("Inserted row count to Students table: " + retInt);
+    error err => io:println("Insert to Students table failed: " + err.message);
 }
-testDB.stop();
 ```
 
-### Inserting data
+In the following second example, parameter values which are in local variables are directly passed as parameters to the `update` operation. This direct parameter passing can be done for any primitive ballerina type like string, int, float, boolean. The sql type of the parameter is derived from the type of the ballerina variable passed in. 
 
-Following are two examples of data insertion by executing an INSERT statement using the `update` operation of the endpoint. 
+```ballerina
+int id = 2;
+string name = "Anne";
+var ret2 = testDB->update("INSERT INTO Students(StudentID, LastName) values (?, ?)", id, name);
+match ret2 {
+    int retInt => io:println("Inserted row count to Students table: " + retInt);
+    error err => io:println("Insert to Students table failed: " + err.message);
+}
+```
 
-In the first example, query parameter values are directly passed into the `update` operation:
+In the following third example, parameter values are passed as `sql:Parameter` to the `update` operation. This sql:Parameter needs to used when we need to provide more details like exact sql type of the parameter, and parameter direction etc. The default parameter direction is "IN". Refer sql package for more details on parameters.
 
 ```ballerina
-//Insert data into Customers table
-var insertCount = check testDB->update("Insert into Customers (firstName,lastName,registrationID,creditLimit,country)
-                                   values ('James', 'Clerk', 2, 5000.75, 'USA')");
-testDB.stop();
+sql:Parameter p1 = { sqlType: sql:TYPE_INTEGER, value: 3 };
+sql:Parameter p2 = { sqlType: sql:TYPE_VARCHAR, value: "James" };
+var ret3 = testDB->update("INSERT INTO Students(StudentID, LastName) values (?, ?)", p1, p2);
+match ret3 {
+    int retInt => io:println("Inserted row count to Students table: " + retInt);
+    error err => io:println("Insert to Students table failed: " + err.message);
+}
 ```
-In the following second example, parameter values are first assigned to local variables of type `sql:Parameter` and then passed into the `update` operation.
+
+### Selecting data
+
+Following is an example of selecting data. First, a type is created to represent the returned result set. Then the SELECT query is execute via the `select` operation of the endpoint by passing that result set type. Once the query is executed, each data record can be retrieved by looping the result set.
```

Comment:
Change "Then the SELECT query is execute" to "Then the SELECT query is executed"